packages/app-folders/README.md

```diff
@@ -21,9 +21,40 @@ yarn add @webiny/app-folders
 ## Usage
```

Comment:
Tip (for future GH docs): when writing GH docs, you can also use the format in this README:
https://github.com/webiny/webiny-js/tree/fix/folder-tree/packages/wcp
https://github.com/webiny/webiny-js/tree/fix/folder-tree/packages/api-wcp
https://github.com/webiny/webiny-js/tree/fix/folder-tree/packages/app-wcp

We [discussed this](https://www.notion.so/webiny/Proposal-Packages-Docs-Reference-f6d9ed56af9a4ae9921e86228bc2d202) a long time ago, and kinda said that we want to use it, but never actually started implementing it. :)